spec/controllers/v1/nominations_controller_spec.rb

```diff
@@ -2,13 +2,18 @@
 
 describe V1::NominationsController,  type: :controller do
   describe "POST create" do
-    context "good args" do
-      it "returns submitted: true" do
+    context "legislator suggestion" do
+      it "submits form" do
+        senator = FactoryGirl.create(:senator, id: 3, first_name: 'Joe', last_name: 'Smith')
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.